dev/TeachingTip/InteractionTests/TeachingTipTestPageElements.cs

```diff
@@ -0,0 +1,417 @@
+﻿using Windows.UI.Xaml.Tests.MUXControls.InteractionTests.Infra;
+using Windows.UI.Xaml.Tests.MUXControls.InteractionTests.Common;
+using System;
+using System.Numerics;
+using Common;
+
+#if USING_TAEF
+using WEX.TestExecution;
+using WEX.TestExecution.Markup;
+using WEX.Logging.Interop;
+#else
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
+#endif
+
+#if BUILD_WINDOWS
+using System.Windows.Automation;
+using MS.Internal.Mita.Foundation;
+using MS.Internal.Mita.Foundation.Controls;
+using MS.Internal.Mita.Foundation.Waiters;
+#else
+using Microsoft.Windows.Apps.Test.Automation;
+using Microsoft.Windows.Apps.Test.Foundation.Controls;
+using Microsoft.Windows.Apps.Test.Foundation.Waiters;
+using Microsoft.Windows.Apps.Test.Foundation;
+#endif
+
+namespace Windows.UI.Xaml.Tests.MUXControls.InteractionTests
+{
+    class TeachingTipTestPageElements
+    {
+        public ListBox GetLstTeachingTipEvents()
+        {
+            if (lstTeachingTipEvents == null)
+            {
+                Log.Comment("Find the lstTeachingTipEvents");
+                lstTeachingTipEvents = new ListBox(FindElement.ByName("lstTeachingTipEvents"));
+                Verify.IsNotNull(lstTeachingTipEvents);
+            }
+            return lstTeachingTipEvents;
+        }
+        private ListBox lstTeachingTipEvents;
+
+        public TextBlock GetEffectivePlacementTextBlock()
```

Comment:
Optional: It would be more idiomatic C# to make these be read-only properties instead of getter methods.